utils/project/sync.go

```diff
@@ -143,3 +154,56 @@ func completeUpload(projectId string, files []string, modfiles []string, timesta
 		// TODO - Need to handle this gracefully.
 	}
 }
+
+func ignoreFileOrDirectory(name string, isDir bool) bool {
+	// List of files not add to PFE
```

Comment:
typo: should read as "List of files not added to PFE" or something similar. Current comment doesn't make sense.

Reply:
Changed to "List of directories that will not be sent to PFE"